packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -226,42 +263,44 @@ export class WorkflowRepository extends Repository<WorkflowEntity> {
 	}
 
 	private applyTagsRelation(qb: SelectQueryBuilder<WorkflowEntity>): void {
-		qb.leftJoin('workflow.tags', 'tags').addSelect(['tags.id', 'tags.name']);
+		qb.leftJoin('workflow.tags', 'tags')
+			.addSelect(['tags.id', 'tags.name'])
+			.addOrderBy('tags.createdAt', 'ASC');
 	}
 
-	private applyOwnedByRelation(qb: SelectQueryBuilder<WorkflowEntity>): void {
-		qb.leftJoin('workflow.shared', 'shared')
-			.addSelect([
-				'shared.role',
-				'shared.createdAt',
-				'shared.updatedAt',
-				'shared.workflowId',
-				'shared.projectId',
-			])
-			.leftJoin('shared.project', 'project')
-			.addSelect([
-				'project.id',
-				'project.name',
-				'project.type',
-				'project.icon',
-				'project.createdAt',
-				'project.updatedAt',
-			]);
+	private applySorting(qb: SelectQueryBuilder<WorkflowEntity>, sortBy?: string): void {
+		if (!sortBy) {
+			this.applyDefaultSorting(qb);
+			return;
+		}
+
+		const [column, direction] = this.parseSortingParams(sortBy);
+		this.applySortingByColumn(qb, column, direction);
 	}
 
-	private applySorting(qb: SelectQueryBuilder<WorkflowEntity>, sortBy?: string): void {
-		if (sortBy) {
-			const [column, order] = sortBy.split(':');
-			const direction = order.toUpperCase() as 'ASC' | 'DESC';
-
-			if (column === 'name') {
-				qb.orderBy(`LOWER(workflow.${column})`, direction);
-			} else {
-				qb.orderBy(`workflow.${column}`, direction);
-			}
-		} else {
-			qb.orderBy('workflow.updatedAt', 'ASC');
+	private parseSortingParams(sortBy: string): [string, 'ASC' | 'DESC'] {
+		const [column, order] = sortBy.split(':');
+		return [column, order.toUpperCase() as 'ASC' | 'DESC'];
+	}
+
+	private applyDefaultSorting(qb: SelectQueryBuilder<WorkflowEntity>): void {
+		qb.orderBy('workflow.updatedAt', 'ASC');
+	}
```

Comment:
Since this is only one line and used only from single place, might make sense to not have this as separate method but instead do it inline

Reply:
Sounds good! Will fix it in a follow up PR, because else I have to wait for the E2E tests to pass again and that takes forever.